ui-tests/src/test/java/org/openmrs/reference/EditPatientRelationshipTest.java

```diff
@@ -0,0 +1,53 @@
+package org.openmrs.reference;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
+import org.openmrs.reference.page.*;
+
+public class EditPatientRelationshipTest  extends ReferenceApplicationTestBase{
+    private static String name = "patients";
```

Comment:
We don't need this test, we rather want to have this written in bdd steps

Reply:
Ohh didnt knew this thanks let me remove this class